xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESink.h

```diff
@@ -63,6 +63,8 @@ class CSinkControlProtocol : public Protocol
     VOLUME,
     FLUSH,
     TIMEOUT,
+    SILENCECOUNTERUPDATED,
```

Comment:
Concerning the second comment: putting it into the streaming message makes no sense, as it is not related.